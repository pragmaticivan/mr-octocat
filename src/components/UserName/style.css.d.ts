export const user_name: string;
export const userName: string;
export const user_github_name: string;
export const userGithubName: string;
