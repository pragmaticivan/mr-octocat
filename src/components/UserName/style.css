.user_name {
  color: #24292e;
  font-size: 26px;
  line-height: 30px;
  font-weight: 600;
  display: block;
}
.user_github_name {
  color: #666666;
  font-size: 20px;
  font-weight: 300;
  line-height: 24px;
  display: block;
}
