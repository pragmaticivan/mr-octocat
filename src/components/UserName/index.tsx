import { Profile } from "../../types";
import React from "react";
import style from "./style.css";

interface Props {
  userProfile: Profile;
}

export default function UserName(props: Props) {
  const { userProfile } = props;
  return (
    <React.Fragment>
      <span className={style.userName}>{userProfile.name}</span>
      <span className={style.userGithubName}>octocat</span>
    </React.Fragment>
  );
}
