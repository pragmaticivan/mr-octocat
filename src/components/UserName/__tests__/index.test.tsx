import UserName from "../";
import React from "react";
import { shallow } from "enzyme";

describe("UserName", () => {
  it("renders the component", () => {
    const profile = {
      avatarUrl: "string",
      url: "string",
      name: "string",
    };
    const wrapper = shallow(<UserName userProfile={profile} />);
    expect(wrapper).toMatchSnapshot();
  });
});
