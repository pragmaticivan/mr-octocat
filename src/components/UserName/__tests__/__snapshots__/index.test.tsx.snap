// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserName renders the component 1`] = `ShallowWrapper {}`;
